mne/filter.py

```diff
@@ -29,7 +29,7 @@ def is_power2(num):
 
     Returns
     -------
-    b : bool
+    bool
```

Comment:
Are you sure about this one? We always name our returned arguments

Reply:
Ah, maybe I have mis-understood the convention here. In the code (which I didn't change any of), the return is anonymous - the literal line is: `return num != 0 and ((num & (num - 1)) == 0)`. I thought in that case the docstring should be un-named, but based on Eric's comment it seems like MNE convention is give a reasonable name in the docstring, even if it's an unnamed return in the code? If so, I can switch this back!

Reply:
yes please switch it back. We always give names to returned values  thx

Reply:
Yes we always name even if anonymous. And even when not anonymous the name we give the user is not necessarily the same as the name in the implementation `return <name>`. So far we mostly try to think of an informative return name.

Reply:
I see, that makes sense, thank you for notes on this! I have switched that change back to how it was!